packages/app/src/settings/project/ProjectId.vue

```diff
@@ -31,11 +31,6 @@
         variant="outline"
       />
     </div>
-    <CloudConnectButton
-      v-else
-      :gql="props.gql"
-      utm-medium="Settings Tab"
-    />
```

Comment:
Kept `v-if="props.gql.currentProject?.projectId"` in the code block above, to prevent complain from TS compiler.

Reply:
Yeah I figured you'd need this haha, thanks!